gh_stats/gh_stats.py

```diff
@@ -62,8 +63,9 @@ def call(self, page_no):
             self.repo,
             self.endpoint,
             "?",
-            self.query({"page": page_no})
+            self.query({"page": page_no, "per_page": 100})
```

Comment:
Ah. Didn’t even see that. (#justhackingaway…)

/Lars

 

From: Gabriel von Heijne [mailto:notifications@github.com] 
Sent: den 2 maj 2016 11:41
To: akvo/akvo-github-stats <akvo-github-stats@noreply.github.com>
Cc: Lars Sjögreen <lars.sjogreen@akvo.org>; Author <author@noreply.github.com>
Subject: Re: [akvo/akvo-github-stats] Ratelimit (#2)

 

In gh_stats/gh_stats.py <https://github.com/akvo/akvo-github-stats/pull/2#discussion_r61720691> :

> @@ -62,8 +63,9 @@ def call(self, page_no):
>              self.repo,
>              self.endpoint,
>              "?",
> -            self.query({"page": page_no})
> +            self.query({"page": page_no, "per_page": 100})

Maybe use the nice PAGESIZE constant?   <https://assets-cdn.github.com/images/icons/emoji/unicode/1f609.png> 

—
You are receiving this because you authored the thread.
Reply to this email directly or  <https://github.com/akvo/akvo-github-stats/pull/2/files/8972dc2e1e72584d47024e1fea313f8bfa04db55#r61720691> view it on GitHub  <https://github.com/notifications/beacon/ARWRe7UWnyYQTHs8VH6SlGtV_1cXPE26ks5p9ca8gaJpZM4ISNDB.gif>